content/reference/admin/private_locations/configuration/gcp/index.md

```diff
@@ -83,3 +94,35 @@ control-plane {
   ]
 }
 ```
+
+## Cloud NAT
+
+Cloud NAT (network address translation) lets certain resources in Google Cloud create outbound connections to the internet or to other Virtual Private Cloud (VPC) networks. 
+Cloud NAT supports address translation for established inbound response packets only. **It does not allow unsolicited inbound connections.**
+
+More info on [Cloud NAT overview](https://cloud.google.com/nat/docs/overview).
+
+### Create instance without external IP
+
+Instance without external IP cannot access internet without a Cloud NAT configured on the network, for the region.
```

Comment:
```suggestion
An instance without an external IP cannot access to Internet without a Cloud NAT configured on its network, for the region.
```

Reply:
question: why "Internet" and not "internet"?